config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   get :dashboard, to: 'companies#dashboard'
-
+  get 'home/index'
```

Comment:
use member and collection blocks for routes

Reply:
done